deploy/charts/secret-manager/values.yaml

```diff
@@ -12,6 +12,36 @@ imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
 
+# Optional ApiServer host address to connect to. If not specified, autoconfiguration will be attempted
+apiServerHost: ""
+# Path to a kubeconfig. Only required if out-of-cluster
+kubeConfig: ""
+# If set, this limits the scope of secret-manager to a single namespace and
+# ClusterSecretStores are disabled. If not specified, all namespaces will be watched
+namespace: ""
+# Leader Election configuration
+leaderConfig:
+  # If true, secret-manager will perform leader election between instances to ensure no more
+  # than one instance of secret-manager operates at a time
+  elect: true
+  # Namespace used to perform leader election. Only used if leader election is enabled
+  namespace: "kube-system"
+  # The duration in seconds that non-leader candidates will wait after observing a leadership
+  # renewal until attempting to acquire leadership of a led but unrenewed leader
+  # slot. This is effectively the maximum duration that a leader can be stopped
+  # before it is replaced by another candidate. This is only applicable if leader
+  # election is enabled.
+  leaseDuration: 60
+  # The interval between attempts by the acting master to renew a leadership slot
+  # before it stops leading. This must be less than or equal to the lease duration.
+  # This is only applicable if leader election is enabled.
+  renewDeadline: 45
+  # The duration the clients should wait between attempting acquisition and renewal
+  # of a leadership. This is only applicable if leader election is enabled.
+  retryPeriod: 15
+# The set of controllers to enable
+defaultEnabledControllers: ""
```

Comment:
This doesnt currently do anything or have plans to do anything, should be stripped from flags and helm chart. This is just left over from copy and paste.